EveryPass/test/mocha_tests.js

```diff
@@ -42,7 +42,7 @@ describe('EveryPass Specific Testing', function () {
                 .setFirefoxOptions(fxoptions)
                 .build();
 
-        await driver.manage().window().fullscreen(); 
+        await driver.manage().window().setRect({ x: 0, y: 0, width: 1920, height: 1080})
```

Comment:
Is this the smallest successful size that can be set? 

I think it should be the smallest successful size so that if the test is run on a monitor that doesn't have the same resolution as yours, the test has a greater chance of succeeding.